ampd/src/broadcaster/confirm_tx.rs

```diff
@@ -47,23 +46,23 @@ impl From<cosmrs::proto::cosmos::base::abci::v1beta1::TxResponse> for TxResponse
 pub enum Error {
     #[error("failed confirming tx due to tx not found: {tx_hash}")]
     Confirmation { tx_hash: String },
-    #[error("failed confirming tx due to grpc error {status}: {tx_hash}")]
-    Grpc { status: Status, tx_hash: String },
+    #[error("failed confirming tx due to grpc error: {tx_hash}")]
```

Comment:
I'm not a fan of using colons in the error string, because that's how the error stacks get concatenated. Can these change to `failed confirming tx {tx_hash}...`?